modules/ROOT/pages/cloudhub-faq.adoc

```diff
@@ -39,6 +39,38 @@ Anypoint Studio can be downloaded https://www.mulesoft.com/lp/dl/studio[here]. F
 
 For specific pricing, please reach out to either your account executive or your account development representative, or contact us in the https://help.mulesoft.com[MuleSoft Help Center].
```

Comment:
```suggestion
For specific pricing, reach out to either your account executive or your account development representative, or contact the https://help.mulesoft.com[MuleSoft Help Center].
```

Reply:
Oh dear...I see what you mean about personal vesting in the words.... I will suggest some changes to avoid "us" and marketese, but understand if it's just not worth it....